sdk/eventhub/event-hubs/src/batchingReceiver.ts

```diff
@@ -188,10 +189,42 @@ export class BatchingReceiver extends EventHubReceiver {
             receiverError
           );
         }
+        if (receiver && !receiver.isItselfClosed()) {
+          if (!this.isConnecting) {
+            log.error(
+              "[%s] 'receiver_close' event occurred on the receiver '%s' with address '%s' " +
+                "and the sdk did not initiate this. The receiver is not reconnecting. Hence, calling " +
+                "detached from the _onAmqpClose() handler.",
```

Comment:
Update `_onAmqpClose` -> `onReceiveClose` in all the log messages that have been added to this file

Reply:
Updated!!